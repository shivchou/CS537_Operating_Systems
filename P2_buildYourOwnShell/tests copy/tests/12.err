Invalid argument passed to history
Invalid argument passed to history
Incorrect usage of history. Correct format: history | history n
