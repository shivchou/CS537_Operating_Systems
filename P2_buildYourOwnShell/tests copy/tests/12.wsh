alias e = echo
e hello
echo world
history
history 2
history 5
history 0
history foo
history 1 2
