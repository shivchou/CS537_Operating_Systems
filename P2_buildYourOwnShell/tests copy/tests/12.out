hello
world
alias e = echo
e hello
echo world
e hello
history 2
